docs/pages/guides/spend-permissions/overview.mdx

```diff
@@ -0,0 +1,80 @@
+# Spend Permissions
+
+Spend Permissions enable third-party signers to spend assets (native and ERC-20 tokens) from user wallets. 
+This unlocks the following use cases: // TODO make note about reduced signing
+- frequent in-game spending
+- subscription payments
+- auto-renewals
+- automated or event-based trading
+- micropayment streaming
+
+Differences between Spend Permissions and ERC-20 permits:
+- Spend Permissions **supports all ERC-20 assets**, whereas permits only support ERC-20s that implement the permit standard  
+- Spend Permissions **enable spending native tokens**  
+- Spend Permissions offers granular **controls for recurrence**  
+- The logic that controls asset movements is **implemented in the user's wallet**, not the asset contract  
+
+
+## The `SpendPermission` details
+A spend permission is defined by the following parameters
+
+```solidity
+struct SpendPermission {
+    address account;
+    address spender;
+    address token;
+    uint160 allowance;
+    uint48 period;
+    uint48 start;
+    uint48 end;
+    uint256 salt;
+    bytes extraData;
+}
+```
+
+
```

Comment:
Consider adding a section for requesting spend permissions? Maybe the details above could into it.